test/string-class-test.js

```diff
@@ -0,0 +1,137 @@
+const chai = require('chai');
+
+const assert = chai.assert;
```

Comment:
**Nitpick**
`chai` is a large library with implementations for `should`, `assert`, and 'expect`. Prefer using [assert](https://nodejs.org/api/assert.html) since that's all you need and not the entire `chai` library.